frontend/src/metabase/components/EntityMenuItem/EntityMenuItem.styled.tsx

```diff
@@ -22,39 +22,46 @@ export const MenuItemContent = styled.div<MenuItemProps>`
     color(props.disabled ? "text-light" : props.color || "text-dark")};
   padding: 0.85em 1.45em;
   text-decoration: none;
+
   :hover {
     color: ${props => color((!props.disabled && props.hoverColor) || "brand")};
     background-color: ${props =>
       !props.disabled && props.hoverBgColor
         ? color(props.hoverBgColor)
         : "var(--mb-color-bg-light)"};
   }
+
   > .Icon {
     color: ${props =>
       color(props.disabled ? "text-light" : props.color || "text-dark")};
     margin-right: 0.65em;
   }
+
   :hover > .Icon {
     color: ${props => color((!props.disabled && props.hoverColor) || "brand")};
-  },
+  }
+
   /* icon specific tweaks
      the alert icon should be optically aligned  with the x-height of the text */
   > .Icon.Icon-alert {
-    transform: translate-y(1px),
```

Comment:
Did this work?

Reply:
It didn't, it's not a valid syntax. Probably safer to drop it

Reply:
[`b2a9499` (#45132)](https://github.com/metabase/metabase/pull/45132/commits/b2a9499bbb060a57978bae05cf0e91bcbf2caf90)